.mvn/extensions.xml

```diff
@@ -0,0 +1,17 @@
+<extensions>
+    <extension>
+        <groupId>com.gradle</groupId>
+        <artifactId>gradle-enterprise-maven-extension</artifactId>
+        <version>1.20.1</version>
+    </extension>
+    <extension>
+        <groupId>com.gradle</groupId>
+        <artifactId>common-custom-user-data-maven-extension</artifactId>
+        <version>1.12.5</version>
+    </extension>
+    <extension>
+        <groupId>org.hibernate.search.develocity</groupId>
+        <artifactId>hibernate-search-develocity-extension</artifactId>
+        <version>1.0.11.Final</version>
+    </extension>
+</extensions>
```

Comment:
I guess these won't be managed by Dependabot, right?

Reply:
I don't think they will. Though I doubt we'll need to update them frequently anyway.

@jprinet suggested that we add the top two as dependencies of `hibernate-search-develocity-extension`, which would somewhat mitigate the problem, at the cost of having to re-release `hibernate-search-develocity-extension` every time we need to upgrade one of these extensions.

Reply:
FYI Dependabot also monitors `.mvn/extensions.xml`.
There is a [compatiblity matrix](https://docs.gradle.com/enterprise/compatibility/#develocity_compatibility_2) between Develocity backend and `gradle-enterprise-maven-extension` though. 

Given that your instance is managed, the time between the extension publication and the backend upgrade is usually minimal. In worst case scenario, you may have some Build Scans not being published if the extension was upgraded to a not yet supported version by the backend.

Hope this clarifies.